src/dsp_tools/utils/excel_to_json_lists.py

```diff
@@ -253,9 +254,8 @@ def validate_lists_section_with_schema(
     """
     if bool(path_to_json_project_file) == bool(lists_section):
         raise BaseError("Validation of the 'lists' section works only if exactly one of the two arguments is given.")
-    current_dir = os.path.dirname(os.path.realpath(__file__))
-    with open(os.path.join(current_dir, "../schemas/lists-only.json")) as schema:
-        lists_schema = json.load(schema)
+    with importlib.resources.files("dsp_tools").joinpath("schemas").joinpath("lists-only.json").open() as schema_file:
```

Comment:
```suggestion
    with files("dsp_tools").joinpath("schemas").joinpath("lists-only.json").open() as schema_file:
```
or is the module name needed here?